setup.py

```diff
@@ -30,7 +30,7 @@
 
 from setuptools import find_packages, setup
 
-package_name = "opensearch"
+package_name = "opensearch-py"
```

Comment:
I think this change needs the [opensearch](https://github.com/opensearch-project/opensearch-py/tree/main/opensearch) directory name also to be changed.

Reply:
> I think this change needs the opensearch directory name also to be changed to opensearch-py.

This would need the whole test suite to be changed. All the `from opensearch` or `import opensearch` would be affected. Not at all suggested.

I think for a quick fix, we can do this.
```py
- with open(join(base_dir, package_name, "_version.py")) as f:
+ with open(join(base_dir, "opensearch", "_version.py")) as f:
```